src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -163,84 +168,156 @@ private void initializeRamseteCommands() {
+              fourBallParallel1,
+              fourBallRamseteCommands[1],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
+              fourBallParallel2,
+              fourBallRamseteCommands[3],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false));
+
+      // ---------------------------------------
+
+      ParallelDeadlineGroup fiveBallParallel1 =
+          new ParallelDeadlineGroup(
+              fiveBallRamseteCommands[0],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      ParallelDeadlineGroup fiveBallParallel2 =
+          new ParallelDeadlineGroup(
+              fiveBallRamseteCommands[2],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      ParallelDeadlineGroup fiveBallParallel3 =
+          new ParallelDeadlineGroup(
+              fiveBallRamseteCommands[4],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      fiveBallCommand =
+          new SequentialCommandGroup(
+              new WaitCommand(initialWaitTime),
+              fiveBallParallel1,
+              fiveBallRamseteCommands[1],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
+              fiveBallParallel2,
+              fiveBallRamseteCommands[3],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
+              fiveBallParallel3,
+              fiveBallRamseteCommands[5],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false));
     }
   }
 
   private void initializeTest() {
     // REPLACE ME to test anything
-    testRamseteCommands = getRamseteCommands(getTrajectories(Constants.Auton.TEST.getPaths()));
-    testCommand =
-        new SequentialCommandGroup(
-            testRamseteCommands[0].andThen(() -> driveBaseSubsystem.stopDriveMotors()),
-            testRamseteCommands[1].andThen(() -> driveBaseSubsystem.stopDriveMotors()));
+    driveTestRamseteCommands = getRamseteCommands(getTrajectories(Constants.Auton.TEST.getPaths()));
+    driveTestCommand = new WaitCommand(initialWaitTime);
+    int i = 0;
+    while (i < driveTestRamseteCommands.length) {
+      driveTestCommand =
+          driveTestCommand.andThen(driveTestRamseteCommands[i]); // adding next ramsete
+      i++;
+    }
   }
 
   public Command getChosenCommand(Constants.Auton mode) {
     switch (mode) {
       case TEST:
         System.out.println("Auton: Test mode selected.");
-        return testCommand;
-      case TAXI:
-        System.out.println("Auton: Taxi mode selected.");
-        return taxiCommand;
+        return driveTestCommand;
+      case PUSHTAXI:
+        System.out.println("Auton: Push Taxi mode selected.");
+        return pushTaxiCommand;
+      case INTAKETAXI:
```

Comment:
What do we mean by push taxi? Is it just taxi without the intake running?

Reply:
It is taxi but the intake is pointed towards the hub so that when it moves out of the tarmac it will push the enemy's ball with the bumper. The intent was so that there would be an auton option if our intake completely wasn't working and we somehow wanted to sabotage the other team.